packages/core/src/utils/make-namespaced.test.ts.md

```diff
@@ -688,3 +688,53 @@ Generated by [AVA](https://avajs.dev).
         },
       },
     }
+
+## make namespaced input - custom layout
+
+> Snapshot 1
+
+    {
+      language: 'Solidity',
+      settings: {
+        evmVersion: 'paris',
+        optimizer: {
+          enabled: true,
+          runs: 200,
+        },
+        outputSelection: {
+          '*': {
+            '': [
+              'ast',
+            ],
+            '*': [
+              'storageLayout',
+            ],
+          },
+        },
+      },
+      sources: {
+        'contracts/test/NamespacedToModifyCustomLayout.sol': {
+          content: `// SPDX-License-Identifier: MIT␊
+          pragma solidity ^0.8.29;␊
+          ␊
+          abstract contract Base {␊
+          }␊
+          ␊
+          abstract contract ExampleCustomLayout  is Base {␊
```

Comment:
There seems to be a space left when the layout is removed. Not a big deal but pointing out if we want to change it

Reply:
The modified contract is only used internally, and whitespace isn't part of the AST so it's not simple to remove and attempting to do so could be more prone to errors.